src/topojson/topojson.ml

```diff
@@ -263,25 +276,27 @@ module Make (J : Intf.Json) = struct
               "A Geojson text should contain one object with a member `type`.")
 
     let rec to_json ?bbox = function
-      | Point point, foreign_members ->
-          Point.to_json ?bbox ~foreign_members point
-      | MultiPoint mp, foreign_members ->
-          MultiPoint.to_json ?bbox ~foreign_members mp
-      | LineString ls, foreign_members ->
-          LineString.to_json ?bbox ~foreign_members ls
-      | MultiLineString mls, foreign_members ->
-          MultiLineString.to_json ?bbox ~foreign_members mls
-      | Polygon p, foreign_members -> Polygon.to_json ?bbox ~foreign_members p
-      | MultiPolygon mp, foreign_members ->
+      | Point point, foreign_members, properties ->
```

Comment:
This is matching a triple of values, but we have (which is good I think) changed to represented the value as a record. So you will need to pull the values out of the record.